mainwindow.py

```diff
@@ -1,10 +1,43 @@
+"""
+Main module for the application.
+
+This module initializes the main application window, sets up the canvas,
+adds sliders for various parameters, and binds events for canvas zooming.
+"""
+
 import tkinter as tk
 
-# Creating main application window
-window=tk.Tk()
-window.title("Fractal Matching Game") 
-window.geometry("600x800") # Set to potrait orientation
-window.resizable(False,False) # Making window non resizable
+from gui import (initialise_gui,
+                 populate_sliders,
+                 bind_canvas_zoom_events)
+from sandbox import draw_squares
```

Comment:
I like that you moved draw_squares into a separate file. I didn't think of doing it, but I also didn't plan to make this an official part of the project. Thanks for such a nice treatment =)

Reply:
I loved them ❤ I guess it needs to removed in the next iteration though, since the tree will be used instead.